ui/apps/platform/src/Containers/SystemHealth/Components/DeclarativeConfigurationHeathCard.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useEffect, ReactElement } from 'react';
+import {
+    Alert,
+    Card,
+    CardBody,
+    CardHeader,
+    CardHeaderMain,
+    CardTitle,
+    Flex,
+    FlexItem,
+} from '@patternfly/react-core';
+import { TableComposable, Tbody, Td, Thead, Th, Tr } from '@patternfly/react-table';
+import { CheckCircleIcon, ExclamationCircleIcon } from '@patternfly/react-icons';
+import pluralize from 'pluralize';
+
+import IconText from 'Components/PatternFly/IconText/IconText';
+import { fetchDeclarativeConfigurationsHealth } from 'services/IntegrationHealthService';
+import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+
+import { IntegrationHealthItem } from '../utils/integrations';
+
+type Props = {
+    pollingCount: number;
+};
+
+function DeclarativeConfigurationHealthCard({ pollingCount }: Props): ReactElement {
+    const [isFetching, setIsFetching] = useState(false);
+    const [requestErrorMessage, setRequestErrorMessage] = useState('');
+    const [items, setItems] = useState<IntegrationHealthItem[]>([]);
+
+    useEffect(() => {
+        setIsFetching(true);
+        fetchDeclarativeConfigurationsHealth()
+            .then((itemsFetched) => {
+                setRequestErrorMessage('');
+                setItems(itemsFetched);
+            })
```

Comment:
nit: I like to clear the error message as soon as a 2nd request is fired off. (Hope springs eternal...)

```suggestion
        setIsFetching(true);
        setRequestErrorMessage('');
        
        fetchDeclarativeConfigurationsHealth()
            .then((itemsFetched) => {
                setItems(itemsFetched);
            })
```

Reply:
See response to next comment.